.github/workflows/build.yml

```diff
@@ -34,7 +34,7 @@ jobs:
 
       - uses: actions/setup-node@v2
         with:
-          node-version: '5.12.0'
+          node-version: '14.15.5'
```

Comment:
🤩

Reply:
This is just CI. We'll need to upgrade `.node-version` and upgrade it in production through ofn-install.